packages/@storefront/structure/test/unit/infinite-list.ts

```diff
@@ -15,4 +15,17 @@ suite('InfiniteList', ({ expect, spy, itShouldProvideAlias }) => {
       expect(infiniteList).to.be.an.instanceOf(List);
     });
   });
+
+  // this test corresponds to a hotfix to a known riot.js bug; this allows `opts._props` to be parsed as `props`
```

Comment:
Tag with `XXX` and move the comment to before the `it`, because it is conceivable that we might have other `onMount` functionality later on.